presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -394,6 +399,12 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
 
     @Override
     public IcebergTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName)
+    {
+        return getTableHandle(session, tableName, Optional.empty(), Optional.empty());
+    }
+
+    @Override
+    public IcebergTableHandle getTableHandle(ConnectorSession session, SchemaTableName tableName, Optional<com.facebook.presto.common.type.Type> timeTravelType, Optional<Object> timeTravelExpression)
```

Comment:
Use a Block for the expression--just a single value block.  Then use Type.getObjectValue to retrieve the value

Reply:
I wouldn't use time travel in the code to describe what we're doing.  I would actually use version.

Reply:
Time travel expression is really just a value, right?